test/e2e/infra/vagrant/playbook/roles/common/templates/containerd.conf.j2

```diff
@@ -0,0 +1,41 @@
+root = "/var/lib/containerd"
```

Comment:
A little unsure how to address this - I've removed the line you mentioned and things still work fine, but I see that there are references online about configuring containerd to run rootless: https://github.com/containerd/containerd/blob/main/docs/rootless.md

Let me know if there's an approach to this that I'm missing

Reply:
I didn't mean to suggest that you should remove that line. rootless is interesting but maybe too much trouble.

I think it's possible to adjust the `uid` / `gid` values in the conf so that the `vagrant` user can run `ctr` without sudo. That's what I had in mind. But I think it can be done in a follow up PR, and it doesn't need to be part of this. Please revert the line removal and I think we can merge this PR.

Reply:
done